### PR TITLE
[IOTSFW-2509] Fix for broken yotta and jenkins builds

### PR DESCRIPTION
**Pull request by hasnainvirk**
This PR will fix the broken yotta and jenkins builds with uvisor

@bogdanm  @AlessandroA

### CODE REVIEW

File: ld/K64FN1M0xxx12.ld [hunk 1]
```diff
@@ -7,7 +7,7 @@ MEMORY
   VECTORS (rx)          : ORIGIN = 0x00000000, LENGTH = 0x00000400
   FLASH_PROTECTION (rx) : ORIGIN = 0x00000400, LENGTH = 0x00000010
   FLASH (rx)            : ORIGIN = 0x00000410, LENGTH = 0x00100000 - 0x00000410
-  RAM (rwx)             : ORIGIN = 0x1FFF0000, LENGTH = 0x00010000 
+  RAM (rwx)             : ORIGIN = 0x1FFF0000, LENGTH = 0x00020000
```
Comment (bogdanm) [inline ld/K64FN1M0xxx12.ld]: That doesn't look right. You're overflowing RAM into RAM2 now, what's the point of this?

**Comment by teetak01** [comment]
+1, at least fixed issues on mbed Client side

**Comment by bogdanm** [comment]
@AlessandroA, can you please take a look at this?

**Comment by hasnainvirk** [comment]
@kjbracey-arm , @AlessandroA , @bogdanm

**Comment by kjbracey** [comment]
We've created a second attempt at this. Have another look.

**Comment by bogdanm** [comment]
Looks much better now, but I'd really like @AlessandroA to look it at (or maybe @meriac if Alessandro is not available).

**Comment by kjbracey** [comment]
I'd like to double-check what the 512 byte reservation is (was?) for, and document it.

Note that there is now a detailed commit message explaining what's happening - GitHub doesn't seem to make this obvious.

**Comment by teetak01** [comment]
+1 this also works for Client

**Comment by ghost** [comment]
The description says Ram contains uvisor.bss followed by stack, but the script itself only appears to place the stack in RAM, everything else goes to RAM2. Is the description wrong or have I mis-read the script?

**Comment by kjbracey** [comment]
Lines 59-80, untouched by this commit, are putting uvisor.bss in RAM.

**Comment by meriac** [comment]
++, LGTM @mjs-arm @hasnainvirk

**Comment by bogdanm** [comment]
Thanks a lot @meriac!

**Pull request merged**